ocs_ci/ocs/utils.py

```diff
@@ -1132,3 +1136,25 @@ def reboot_node(ceph_node, timeout=300):
     except SSHException:
         log.exception(f"Failed to connect to node {ceph_node.hostname}")
         raise
+
+
+def verify_all_nodes_created():
+    """
+    Verify all nodes are created or not
+
+    Raises:
+        NotAllNodesCreated: In case all nodes are not created
+
+    """
+    # importing here to avoid circular import
+    from ocs_ci.ocs.node import get_all_nodes
```

Comment:
What about to move this function to ocs_ci.ocs.node so you will not have the issues with cyclical import and in the place where you would like to use it you will import from ocs_ci.ocs.node directly.

Reply:
Done